zmon-controller-ui/js/controllers/EntityCtrl.js

```diff
@@ -17,6 +17,53 @@ angular.module('zmon2App').controller('EntityCtrl', ['$scope', '$window', '$rout
             var timeIntervalSinceLastUpdate = MainAlertService.millisecondsApart(epochPastTs, MainAlertService.getLastUpdate());
             return timespanFilter(timeIntervalSinceLastUpdate);
         };
+
+        $scope.numberNiceFormat = function(v) {
+            var ndx = Math.floor( Math.log(v) / Math.log(1000) );
+            var suffix = ["", "K", "M", "G", "T", "P"];
```

Comment:
please use SI unit prefixes, i.e. "k" for kilo: https://en.wikipedia.org/wiki/Metric_prefix